src/cluster_legacy.c

```diff
@@ -2995,13 +2998,13 @@ int clusterProcessPacket(clusterLink *link) {
          * flags, replicaof pointer, and so forth, as this details will be
          * resolved when we'll receive PONGs from the node. */
         if (!sender && type == CLUSTERMSG_TYPE_MEET) {
-            clusterNode *node;
+            clusterNode *n;
 
-            node = createClusterNode(NULL, CLUSTER_NODE_HANDSHAKE);
-            serverAssert(nodeIp2String(node->ip, link, hdr->myip) == C_OK);
-            getClientPortFromClusterMsg(hdr, &node->tls_port, &node->tcp_port);
-            node->cport = ntohs(hdr->cport);
-            clusterAddNode(node);
+            n = createClusterNode(NULL, CLUSTER_NODE_HANDSHAKE);
+            serverAssert(nodeIp2String(n->ip, link, hdr->myip) == C_OK);
+            getClientPortFromClusterMsg(hdr, &n->tls_port, &n->tcp_port);
+            n->cport = ntohs(hdr->cport);
+            clusterAddNode(n);
```

Comment:
Do we need to update these?

Reply:
strictly speaking no and so are the remaining name changes. The point that I am trying to get across is "does it help?". If folks believe readability gets worse, I will revert. Otherwise, I just don't see when we get to improve the readability. It is not like we would dedicate PRs just for improving readability :-)

Reply:
Would have preferred `node` while writing from scratch. Working around Redis/Valkey has made me not modify lines not related to my PR.

Would let the approvers decide. I would prefer merging this out :)

Reply:
> It is not like we would dedicate PRs just for improving readability :-)

I would merge a PR that just makes readability better, I also think you would too, so I don't think this is true?

Reply:
>so I don't think this is true?

Yes I do think it helps and we have used short names for temps pretty extensively in the code base, especially `n`. that said, I don't want to block the more important work on such a trivial thing. I reverted it back. Let's merge and move on?